### DISCUSSION TITLE
7.0.0

### DISCUSSION THREAD

**Discussion by danvick** [danvick/flutter_form_builder#900] [General]
**BREAKING CHANGE**:
* For ease of maintainability, validation functionality has been broken up into a separate package: [form_builder_validators](https://pub.dev/packages/form_builder_validators)

<hr /><em>This discussion was created from the release <a href='https://github.com/danvick/flutter_form_builder/releases/tag/7.0.0'>7.0.0</a>.</em>

**Comment by LiangHuangBC** [comment]
Tried 7.0, and our app crashed, a lot changes need to be done if we want to keep 7.0, so revert back to 6......

**Comment by ashim-kr-saha** [comment]
@danvick It will be very helpful if you release 7.2.0 in [https://pub.dev/packages/flutter_form_builder](https://pub.dev/packages/flutter_form_builder) also.

**Comment by LiangHuangBC** [comment]
Upgrade to core 7.0 with validator 7.2, looks good now but dependency of color picker is old version, conflict with latest version. Have to down grade color picker.
